backend/game/player/test_player.py

```diff
@@ -139,4 +154,4 @@ def test_remove_card(player):
 def lose_all_stars(self):
     player.update_victory_points_by(10)
     player.lose_all_stars()
-    assert player.victory_points == constants.DEATH_HIT_POINT
\ No newline at end of file
+    assert player.victory_points == constants.DEATH_HIT_POINT
```

Comment:
instead of `constants.DEATH_HIT_POINT` just use 0 or make a new constant. Personally, I don't think there should be a constant for 0

Reply:
this was actually not changed confirm its from master but remember he wanted constants used for this test last quarter